src/container.py

```diff
@@ -6,7 +6,7 @@
 
 from typing import TextIO
 
-from charms.vault_k8s.v0.vault_tls import WorkloadBase
+from charms.vault_k8s.v0.vault_managers import WorkloadBase
```

Comment:
Why do we import this from the vault managers? It feels strange for container.py to depend on Vault managers (or on Vault TLS for that matter).

I understand this is not quite part of this change here so feel free to ignore my comment.